codex-rs/rmcp-client/Cargo.toml

```diff
@@ -16,6 +16,15 @@ rmcp = { version = "0.7.0", default-features = false, features = [
     "schemars",
     "server",
     "transport-child-process",
+    "transport-streamable-http-client-reqwest",
```

Comment:
Note the other `Cargo.toml` files have been updated to use `{ workspace = true }`